core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/SubmissionApiServiceSpec.scala

```diff
@@ -143,14 +143,12 @@ class SubmissionApiServiceSpec extends ApiServiceSpec {
               assertResult(StatusCodes.OK) {
                 status
               }
-              return responseAs[SubmissionStatusResponse]
+              responseAs[SubmissionStatusResponse]
             }
         }
     } { capturedMetrics =>
       capturedMetrics should contain (expectedSubmissionStatusMetric(wsName, SubmissionStatuses.Submitted, 1))
     }
-
-    fail("Unable to create and monitor submissions")
```

Comment:
It returns the `SubmissionStatusResponse` as it did before.  Tests which consume that response can determine if it's correct so it's not necessary to force a fail.

Also, the `capturedMetrics` check actually runs now.